focus-android/app/src/main/res/values/strings.xml

```diff
@@ -481,6 +481,9 @@
     <!-- Snackbar action to immediately open the successfully downloaded file. -->
     <string name="download_snackbar_open">Open</string>
 
+    <!-- Snackbar that will be displayed after a website has been added to shortcut. -->
+    <string name="added_to_shortcut">Added to shortcuts</string>
```

Comment:
Let's update this string's ID and it's content to match Fenix:

`<string name="snackbar_added_to_shortcuts">Added to shortcuts!</string>`